source/app/service-providers/assets-provider.ts

```diff
@@ -77,6 +80,13 @@ export default class AssetsProvider {
         return await this.listSnippets()
       } else if (command === 'rename-snippet') {
         return await this.renameSnippet(payload.name, payload.newName)
+      } else if (command === 'getLatexCommands') {
+        try {
+          await fs.access(this._latexCommandsFile, fsConstants.F_OK)
```

Comment:
Up until now I've been using `fs.lstat` to check. Is there any benefit in using `access` rather than `lstat`? If so, we may want to switch the rest of the codebase gradually :D

Reply:
Shouldn't matter much, at least after some quick googling. https://stackoverflow.com/questions/54514627/nodejs-fs-stat-or-fs-access-to-check-if-a-folder-exists has some discussion, and access seems to be preferred and is maybe a bit faster; but both should be fine.